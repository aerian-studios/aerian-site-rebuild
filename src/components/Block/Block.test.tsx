/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

import { Block } from "./index";


describe("Block", () => (
    it("renders correctly", () => {
        const tree = renderer
            .create(<Block className="myClass" />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
));