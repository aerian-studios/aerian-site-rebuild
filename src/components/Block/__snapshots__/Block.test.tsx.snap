// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block renders correctly 1`] = `
<div
  className=" myClass"
/>
`;
